KBKeyboardObserver/KBKeyboardObserver.m

```diff
@@ -104,6 +144,72 @@ - (void)keyboardDidHide:(NSNotification*)aNotification
     }
 }
 
+- (void)keyboardWillChangeFrame:(NSNotification*)aNotification
+{
+    if (self.interfaceRotation)
+    {
+        return;
+    }
+    
+    NSDictionary *info = aNotification.userInfo;
+    CGRect kbRectEnd = [info[UIKeyboardFrameEndUserInfoKey] CGRectValue];
+    NSTimeInterval duration = [info[UIKeyboardAnimationDurationUserInfoKey] doubleValue] ;
+    CGRect viewKbRectEnd = [self.referenceView convertRect:kbRectEnd fromView:nil];
+    
+    
+    if(CGRectContainsRect(self.referenceView.frame, viewKbRectEnd) && !self.standardKeyboard) {
```

Comment:
What is the purpose of this whole if else? `keyboardWillHide:` and `keyboardWillShow:` methods already handles those delegate calls.